src/client/Microsoft.Identity.Client/Internal/Requests/AuthenticationRequestParameters.cs

```diff
@@ -86,6 +86,8 @@ internal class AuthenticationRequestParameters
 
         public string ClaimsAndClientCapabilities { get; private set; }
 
+        public string SuggestedCacheKey { get; set; }
```

Comment:
Logically this does not belong in this class. Let me have if we can put it somewhere else...